pkg/consts/views.go

```diff
@@ -165,6 +165,26 @@ function(doc) {
 }`,
 }
 
+// SharedDocsByDocType is the view for fetching a list of sharingid
+// associated with a docType. The reduce function is to avoid duplicates
+var SharedDocsByDocType = &couchdb.View{
```

Comment:
I don't think you need a view. You can query _all_docs with `start_key: "io.cozy.files/", end_key: "io.cozy.files/\uFFFF", include_docs: true`